app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js

```diff
@@ -137,7 +137,7 @@ define([
 
                         if (eventData.redirectParameters.length > 0) {
                             parameters = res.backUrl.split('#');
-                            parameters.push(eventData.redirectParameters.join('&'));
```

Comment:
Hi, @VDCProjects. Thanks for collaboration. Looks like you removed usage of redirectParameters at all, it's not correct because in other use cases this functionality will break.